jenkins/e2e-image/e2e-runner.sh

```diff
@@ -273,25 +327,47 @@ EXIT_CODE=0
 # We get the Kubernetes tarballs unless we are going to use old ones
 if [[ "${JENKINS_USE_EXISTING_BINARIES:-}" =~ ^[yY]$ ]]; then
```

Comment:
If we keep this in bash I would like to see unit tests that validate various sets of environment variables copy files from X and into Y.